Ports/NiosII/GNU/os_cpu_c.c

```diff
@@ -0,0 +1,389 @@
+/*
+*********************************************************************************************************
```

Comment:
add 17 '*' to the end of line to match the function comment blocks and center align